OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -25,6 +25,9 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 {
 	public class LobbyLogic : ChromeLogic
 	{
+		const string TeamDropdownWidgetName = "TEAM_DROPDOWN";
```

Comment:
Defining these as constants is inconsistent with the rest of this file (and all of the other chrome logic), so this actually hurts readability.  Please keep these inline.